components/footer/Footer.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  FAUCETS,
+  FooterLinkInfo,
+  LEGAL,
+  NETWORKS,
+  RESOURCES,
+  SDKs,
+  SOLUTIONS,
+} from "./footerLinks";
+import { SOCIALS } from "./socialLinks";
+import {
+  Box,
+  ButtonGroup,
+  Container,
+  Flex,
+  SimpleGrid,
+  Stack,
+} from "@chakra-ui/react";
+import { Logo } from "components/logo";
+import { PRODUCTS } from "components/product-pages/common/nav/DesktopMenu";
+import {
+  Heading,
+  LinkButton,
+  TrackedIconButton,
+  TrackedLink,
+} from "tw-components";
+
+type FooterLinkGroupProps = {
+  heading: string;
+  links: FooterLinkInfo[];
+};
+
+function FooterLinkGroup(props: FooterLinkGroupProps) {
```

Comment:
generally prefer `const FooterLinkGroup: React.FC<FooterLinkGroupProps> = (props) => ...` syntax

Reply:
@jnsdls There are some good arguments against using `React.FC` - https://github.com/facebook/create-react-app/pull/8177 but I'm with it either way - so changing all to use `React.FC` as per your suggestion to keep it consistent with rest of the codebase

Reply:
Done in https://github.com/thirdweb-dev/dashboard/pull/716/commits/d3d97b55cb9396a8c2ed3cac6240cf8c8e336b60

Reply:
thx, just as a heads up a lot of the feedback in that thread is no longer up to date (children and generics for example) though I agree there are trade-offs here - for now we'll stick with what we're doing in the project so thanks for the update!